app/auth/views.py

```diff
@@ -2,19 +2,19 @@
 import os
 import re
 from flask.views import MethodView
-from flask import make_response, request, jsonify
+from flask import make_response, request, jsonify, render_template, url_for
 from app.models import User, BlacklistToken
 from flask_bcrypt import Bcrypt
-from flask_mail import Mail, Message
+from flask_mail import Message
+from itsdangerous import URLSafeTimedSerializer, SignatureExpired
 
-from app import create_app
+from run import MAIL
```

Comment:
why did you consider instantiating Mail in `run.py`? an alternative would be to instantiate it in `init`, like `mail = Mail()` before the `create_app` function, then initialize it inside the function using `mail._init_app(app)`

Reply:
This is a good Idea. I will implement it this way and test it out